README.md

```diff
@@ -1,9 +1,31 @@
-[![Build Status](https://travis-ci.org/quran/quran_android.svg?branch=master)](https://travis-ci.org/quran/quran_android)
+<div align="center">
+
+<img src="https://raw.githubusercontent.com/quran/quran_android/master/app/src/madani/res/drawable-xxhdpi/icon.png" alt='Quran for Android logo'/>
+
 # Quran for Android
-[<img align="right" alt="Get it on Google Play" height="128" src="https://play.google.com/intl/en_us/badges/images/generic/en_badge_web_generic.png">](https://play.google.com/store/apps/details?id=com.quran.labs.androidquran)
+
+[![Build Status](https://travis-ci.org/quran/quran_android.svg?branch=master)](https://travis-ci.org/quran/quran_android)
+[![Version](https://img.shields.io/github/v/release/quran/quran_android?include_prereleases&sort=semver)](https://github.com/quran/quran_android/releases/latest)
+[![IzzyOnDroid Version](https://img.shields.io/endpoint?url=https://apt.izzysoft.de/fdroid/api/v1/shield/com.quran.labs.androidquran)](https://apt.izzysoft.de/fdroid/index/apk/com.quran.labs.androidquran)
```

Comment:
I have no idea who this izzy is 😂 maybe we can drop these next 2 urls and just keep the GitHub ones?

Reply:
An f-droid repo. Learnt about it from https://github.com/quran/quran_android/issues/755#issuecomment-927292714.

You mean drop IzzyOnDroid links?

Reply:
yes please